axiom/item.py

```diff
@@ -1135,3 +1137,17 @@ class _PowerupConnector(Item):
 
 POWERUP_BEFORE = 1              # Priority for 'high' priority powerups.
 POWERUP_AFTER = -1              # Priority for 'low' priority powerups.
+
+
+
+def empowerment(iface, priority=0):
+    """
+    Class decorator for indicating a powerup's powerup interfaces.
```

Comment:
I think this documentation should be a little more explanatory, particularly that it also declares the thing it’s decorating as implementing the interface.

Reply:
And document the parameters.

Reply:
Done.